src/test/suite/didact.test.ts

```diff
@@ -168,7 +179,13 @@ suite('Didact test suite', () => {
 	});
 
 	test('test the command line requirement return checking', async () => {
-		const href = testReq;
+		let href = '';
+		const osName = process.platform;
+		if (osName.startsWith(OS_WINDOWS)) {
+			href = testReqWin;
+		} else if (osName.startsWith(OS_LINUX)) {
+			href = testReq;
+		}
```

Comment:
can you explain why there are 2 different cases for windows and linux?

Reply:
because `uname` doesn't exist on windows

Reply:
so for the test, `uname` is used on Linux and `echo test` on windows?
What about using `echo test` in both cases to use the same command for the test?

Reply:
isn't it better to test alternative options as well? I could get away with echo on all fronts, but I think it's better to make sure we're not putting all our eggs in one basket, so to speak

Reply:
I went ahead and changed it to be more consistent. We are testing os-dependent cli in another test "test the command line requirements checking" so we should be ok

Reply:
I think the added value of testing the same command on different OS is higher than different ones.
if the two commands have different structures that is interesting to test, it will be interesting to have 2 tests then. And ensure that it is played on the 2 OSes.
not a blocker for the PR.